app/views/help_center/articles/contents/_74-the-analytics-dashboard.html.erb

```diff
@@ -40,6 +41,28 @@
   <p>A line graph of sales volume is overlaid on top to give you a sense of sales volume relative to your views.</p>
   <p>Hover on any day to see more information about the sales, views, and absolute sales value in dollars.</p>
   <figure><%= image_tag "https://d33v4339jhl8k0.cloudfront.net/docs/assets/5c4657ad2c7d3a66e32d763f/images/6246932342ba434a7afe2278/file-V5ESJfaXNp.png" %></figure>
+  <h3 id="churn">Churn</h3>
```

Comment:
This is how is being rendered:

<img width="3332" height="2718" alt="CleanShot 2025-12-28 at 10 13 30@2x" src="https://github.com/user-attachments/assets/740bbb56-b4c6-4a2a-99af-51a9eeb9e301" />